pennylane/transforms/zx/converter.py

```diff
@@ -337,7 +337,7 @@ def processing_fn(res):
             else:
                 expanded_operations.append(op)
 
-        expanded_tape = QuantumScript(expanded_operations, mapped_tape.measurements, [])
+        expanded_tape = QuantumScript(expanded_operations, mapped_tape.measurements)
```

Comment:
technically a real change, just removed the empty `prep` to avoid raising a warning